__init__.py

```diff
@@ -16,12 +16,17 @@
         UserWarning,
     )
 
-if not os.getenv("HDF5_PLUGIN_PATH"):
-    # Set up the plugin path for HDF5 to pick up compression plugins.
-    plugin_path = libtbx.env.under_base(os.path.join("lib", "plugins"))
+plugin_path = libtbx.env.under_base(os.path.join("lib", "plugins"))
```

Comment:
`legacy_plugin_path` & `plugin_path` with a view to removing the legacy version in ${TBD}